libs/ngworker/lumberjack/http-driver/src/lib/http-driver.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { HttpRequest } from '@angular/common/http';
+import { HttpClientTestingModule, HttpTestingController, TestRequest } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+
+import { resolveDependency } from '@internal/test-util';
+import { LogDriver, LogDriverToken, LumberjackLogLevel, LumberjackModule } from '@ngworker/lumberjack';
+
+import { HttpDriverOptions } from './http-driver-options';
+import { HttpDriverModule } from './http-driver.module';
+import { HttpLogEntry } from './http-log-entry';
+import { HttpDriver } from './http.driver';
+
+/**
+ *
+ * Make given times calls to a function
+ *
+ */
+function repeat(times: number, fn: () => void): void {
+  return Array(times).fill(undefined).forEach(fn);
```

Comment:
Remove `return` and add `new` to prevent confusion (event though it isn't necessary).
```suggestion
  new Array(times).fill(undefined).forEach(fn);
```